tsdb/docs/format/wal.md

```diff
@@ -85,4 +85,34 @@ and specify an interval for which samples of a series got deleted.
 │ └─────────┴───────────────────┴───────────────────┘ │
 │                        . . .                        │
 └─────────────────────────────────────────────────────┘
+<<<<<<< HEAD
+=======
 ```
+
+### Exemplars records
+
+Exemplars records encode exemplars as a list of quadruples `(series_id, timestamp, max_time, exemplar)`.
+Series reference and timestamp are encoded as deltas w.r.t the first sample.
+The first row stores the starting id and the starting timestamp.
```

Comment:
Can we get rid of this first row? The first examplar could have the full ID and the timestamp, and remaining examplars to have the delta.

Reply:
Yeah so this is my only concern with the format. We can skip the first `│ id <8b> │ timestamp <8b> │` and instead have the full id and timestamp for the first exemplar and delta from the 2nd exemplar onewards.

Reply:
👍🏽

Reply:
What are the benefits from this change?  The current implementation to me has the following benefits:  improved readability since baseline and delta values are separate, matches samples, and simpler logic since parsing an exemplar doesn't differ for index 0 versus 1+.

Reply:
Revisiting this again: There is not benefit in this, it just looks more compact. But I just took a look at the [samples record](https://github.com/prometheus/prometheus/blob/main/tsdb/docs/format/wal.md#sample-records) and I think we can go ahead with this with explicit id and timestamp to be consistent. You can ignore my comment.

Reply:
Ack, let's resolve this then?